app/celery/celery.py

```diff
@@ -1,8 +1,21 @@
 import logging
 import time
-
-from celery import Celery, signals, Task
-from celery.signals import task_prerun, task_postrun, worker_process_shutdown, worker_shutting_down, worker_process_init
+from typing import Any
+
+from celery import Celery, Task
+from celery.worker.request import Request
```

Comment:
these should be in alphabetical order, so `from celery.worker.request import` should go below `from celery.signals import`, please.